milabench/cli.py

```diff
@@ -199,6 +207,9 @@ def _get_multipack(
     now = str(datetime.today()).replace(" ", "_")
     run_name = run_name.format(time=now)
 
+    system_config = build_system_config(system_config_path)
+    overrides = merge(overrides, {"system": system_config})
```

Comment:
I would merge it the other way around so that --override can override values in the system config. Also, you might have to set `{"*": {"system": system_config}}` instead, in order to add `system` to every benchmark.